docs/04_web/01_basics/00_prerequisites.mdx

```diff
@@ -47,15 +47,14 @@ If you get confused by any of the terminology we use in this section, we are pro
 - [Building and using components](https://www.fast.design/docs/category/building-components)
 - [Using CSS](https://www.fast.design/docs/fast-element/leveraging-css)
 - [Design tokens](https://www.fast.design/docs/design-systems/design-tokens)
-- [How to style existing components](../../../web/basics/inserting-a-grid/#going-further)
```

Comment:
I would rather not have to get rid of this line, but I can't find a suitible place to link it to since all the docs were changed. I think @wjhendry is looking if there is anything I can use though